src/components/sponsors_tier.tsx

```diff
@@ -0,0 +1,231 @@
+    useState<ContentItem[]>(goldContentData);
+  const [transitionDirection, setTransitionDirection] = useState<
+    "left" | "right" | "newCard" | ""
+  >("");
+
+  const handleTransitionEnd = () => {
+    setIsTransitioning(false);
+    setTransitionDirection("");
+  };
+
+  const handleNavigation = (direction: "prev" | "next") => {
+    setIsTransitioning(true);
+
+    if (direction === "prev") {
+      setCurrentContent((prev) =>
+        prev > 0 ? prev - 1 : currentContentData.length - 1
+      );
+      setTransitionDirection("left");
+    } else {
+      setCurrentContent((prev) =>
+        prev < currentContentData.length - 1 ? prev + 1 : 0
+      );
+      setTransitionDirection("right");
+    }
+  };
+
+  const handleContentChange = (contentType: "gold" | "silver" | "bronze") => {
+    setIsTransitioning(true);
+
+    switch (contentType) {
+      case "gold":
+        setCurrentContentData(goldContentData);
+        break;
+      case "silver":
+        setCurrentContentData(silverContentData);
+        break;
+      case "bronze":
+        setCurrentContentData(bronzeContentData);
+        break;
+      default:
+        setCurrentContentData(goldContentData);
+    }
+
+    // Reset currentContent to the first item in the new content data
+    setCurrentContent(0);
+    setTransitionDirection("newCard");
+  };
+
+  const { image, header, subheader, text, link } =
+    currentContentData[currentContent];
+
+  return (
+    <div className="flex flex-col sm:flex-row rounded-xl shadow-sm bg-gray-300 border-gray-700 h-[750px] md:h-[500px]">
+      {/* DNA Pic */}
+      <div className="flex flex-row flex-shrink-0 w-full rounded-t-xl sm:rounded-s-xl sm:w-1/4 md:rounded-se-none h-auto bg-emerald-500 items-center">
+        <div className="flex md:flex-col flex-row m-auto md:w-28 md:h-80 w-56 h-16 my-3 bg-red-500 content-center justify-around">
+          {/* Navigation buttons for different sponsor types */}
+          <button
+            className="text-white"
+            onClick={() => handleContentChange("gold")}
+          >
+            Gold Sponsors
+          </button>
+          <button
+            className="text-white"
+            onClick={() => handleContentChange("silver")}
+          >
+            Silver Sponsors
+          </button>
+          <button
+            className="text-white"
+            onClick={() => handleContentChange("bronze")}
+          >
+            Bronze Sponsors
+          </button>
+        </div>
+      </div>
+
+      {/* Right Content */}
+      <div className="flex w-full sm:px-5 items-center  my-auto">
+        <div className="p-4 flex flex-col h-full w-full sm:p-7">
+          <div className="flex flex-row items-center justify-between h-[600px] sm:h-[500px] md:h-[350px]">
+            <button
+              className="nav-button sm:mx-10 mr-3"
+              onClick={() => handleNavigation("prev")}
+            >
+              &lt;
+            </button>
```

Comment:
Please use icon button that comes with pre-line (for standardized style)
https://preline.co/docs/buttons.html#icon